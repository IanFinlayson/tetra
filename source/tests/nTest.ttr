def main():
   i = 12
   print("In main(), just set i = 12 and i = ", i, "\n")
   if i == 12:
      i = 10
      print("In if block and just set i = 10, now i = ", i, "\n")

