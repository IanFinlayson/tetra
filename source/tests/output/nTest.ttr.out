In main(), just set i = 12 and i = 12
In if block and just set i = 10, now i = 10

